serverlessworkflow/modules/ROOT/pages/tooling/serverless-workflow-editor/swf-editor-overview.adoc

```diff
@@ -1,8 +1,6 @@
 = Kogito Serverless Workflow Editor
 
-The Serverless Workflow Editor is a tool to manipulate the content of serverless workflow files and simultaneously view the workflow diagram.
-The editor has two parts, a code editor with the source code on the left and a workflow diagram visualizer on the right.
-As you make changes to the content of the editor, the diagram gets updated with the new change.
+The Kogito Serverless Workflow Editor is a tool to manipulate the content of CNCF Serverless Workflow Specification files and simultaneously view the workflow diagram. The editor has two parts, a code editor with the source code on the left and a workflow diagram visualizer on the right. As you make changes to the content of the editor, the diagram gets updated with the new change.
```

Comment:
Please add the link to "CNCF Serverless Workflow Specification". Every time we mention a project at the beginning of the guide, we should add a link back to it.